ntbs-integration-tests/TransferPages/ActionTransferPageTests.cs

```diff
@@ -76,5 +103,33 @@ public async Task AcceptTransferAlert_SuccessfullyChangesTbServiceOfNotification
             Assert.Contains("TestCase TestManager", overviewPage.QuerySelector("#banner-case-manager").TextContent);
             Assert.Null(overviewPage.QuerySelector("#alert-3"));
         }
+
+
+        // This is untested as currently we are unable to add alerts to the in-memory database, we can only seed and remove them.

```

Comment:
Test mentioned in the description of the PR where it tries to add a TransferRejectedAlert and the context seems happy when we add it then forgets it exists as soon as we navigate away from the AddAlert method in the repo

Reply:
@jasiekmiko this seems to be an issue with the in-memory database - @ChristianMcc have you had a chance to google around it?

Reply:
I've googled a fair amount to no avail sadly, will give it one more shot today